lib/registrant.rb

```diff
@@ -0,0 +1,23 @@
+class Registrant
+  attr_reader :name, :age, :permit, :license_data
+  
+  
+  def initialize(name, age, permit = false)
+    @name = name
+    @age = age
+    @permit = permit
+    @license_data = {
+      :written => false,
+      :license => false,
+      :renewed => false,
+      }
+  end
+
+  def permit?   
+    @permit
+  end
+
+  def earn_permit
+    @permit = true
```

Comment:
You can make this even more robust by adding a check that `@permit = true if @age >= 15`.